### ISSUE TITLE
Property 'removeListener' does not exist on type 'EventEmitter'. Did you mean 'removeAllListeners'?

### ISSUE THREAD

**Issue by Kegovc** [bastengao/react-native-usb-serialport-for-android#1]
please change method "close"
```
close(): Promise<any> {
    for (const listener of this.listeners) {
      this.eventEmitter.removeListener(DataReceivedEvent, listener);
    }
    return UsbSerialportForAndroid.close(this.deviceId);
  }
```
by
```
close(): Promise<any> {
    if (this.listeners.length) {
      this.eventEmitter.removeAllListeners(DataReceivedEvent);
    }
    return UsbSerialportForAndroid.close(this.deviceId);
  }
```

**Comment by bastengao** [comment]
@Kegovc Which version of react-native do you use?

**Comment by bastengao** [comment]
I released v0.4.1, you can try it.
